.github/workflows/PR_build_extension.yml

```
@@ -53,7 +53,7 @@ jobs:
       - name: Copy language-server dist directory inside vscode package
         run: |
           cp -R packages/language-server packages/vscode
-          rm -R packages/vscode/language-server/src packages/vscode/language-server/test
+          rm -R packages/vscode/language-server/src packages/vscode/language-server/src/__test__
```

```suggestion
          rm -R packages/vscode/language-server/src
```
This should do, now that I read it completely 😄

---

It already deletes `packages/vscode/language-server/src` so nothing else is needed

---

Ah cool!

---

That also explains why the build was failing, it's already deleted so ofc it can't find it